src/main/java/io/vertx/core/eventbus/EventBus.java

```diff
@@ -261,16 +261,35 @@
    *
    * @param interceptor  the interceptor
    * @return a reference to this, so the API can be used fluently
+   * @deprecated use {@link EventBus#addInterceptor(Interceptor)} instead.
    */
+  @Deprecated
   EventBus addInterceptor(Handler<SendContext> interceptor);
```

Comment:
Ok. I'll likely have to go and update the existing interceptors, though, which shouldn't be an issue.